redis_broker.go

```diff
@@ -52,6 +52,11 @@ func (cb *RedisCeleryBroker) SendCeleryMessage(message *CeleryMessage) error {
 	return nil
 }
 
+//Set custom queue name
```

Comment:
this will break lint

Reply:
Thanks for your review.
Do you mean like this？
`// RedisCeleryBroker is celery broker for redis
type RedisCeleryBroker struct {
	*redis.Pool
	QueueName string
}

// NewRedisBroker creates new RedisCeleryBroker with given redis connection pool
func NewRedisBroker(conn *redis.Pool, queueName string) *RedisCeleryBroker {
	return &RedisCeleryBroker{
		Pool:      conn,
		QueueName: queueName,
	}
}`

Reply:
> this will break lint

Reply:
yes